kernelci/rootfs.py

```diff
@@ -58,9 +73,10 @@ def build(self, config, data_path, arch):
                 for key, value in debos_params.items()
             )
         )
-        cmd = "cd {path} && debos --memory=4G {opts} rootfs.yaml".format(
-            path=data_path, opts=debos_opts
-        )
+        cmd = f"debos --memory=4G {debos_opts}"\
+            f" --artifactdir={absoutput_dir} "\
+            f"{self.kci_path}/config/rootfs/debos/rootfs.yaml"
```

Comment:
One issue is that we can't rely on the config files to always be in that sub-directory. This works when running `kci_rootfs` from a plain git checkout of kernelci-core, but if the package is installed then `kci_rootfs` may be in `/usr/local/bin` and the configuration files in `/etc/` or `/usr/local/share`. We don't have this problem with how Jenkins jobs are currently run, but if we lose the ability to specify where the config files are and where the build output should be then we'll most likely have to add it again at some point. So it's better if we can keep support for that without complicating the code. If it's an issue then we can simplify things and only make it work with the Jenkins use-case for now.

Reply:
Also, `os.path.join()` is the recommended way of constructing file paths.

Reply:
Ok, will put back this option